unified-codes/apps/web/src/components/containers/detail/DetailEntityList.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import { IEntity } from '@unified-codes/data';
+import { List } from '@unified-codes/ui/components';
+import { createStyles, makeStyles } from '@unified-codes/ui/styles';
+
+import DetailEntityListItem from './DetailEntityListItem';
+
+import { IState } from '../../../types';
+import { DetailSelectors } from '../../../selectors';
+import { ITheme } from '../../../styles';
+
+const useStyles = makeStyles((_: ITheme) => createStyles({
+    root: {
+        width: '100%',
+    }
+}));
+
+export interface DetailEntityListProps {
+    entity: IEntity;
+}
+
+export type DetailEntityList = React.FunctionComponent<DetailEntityListProps>;
+
+export const DetailEntityListComponent: DetailEntityList = ({ entity }) => {
+    const classes = useStyles();
+
+    return (
```

Comment:
Would be nice to have a 'no results' option here too - if there are no children to display

Reply:
For now just not rendering, but will need to address this in a separate issue (a bit tricky to handle these sort of cases with recursion... i.e. we don't want the leafs also displaying "no results", so will have to think about this one a bit).